deluge/ui/web/server.py

```diff
@@ -536,6 +585,28 @@ def __init__(self):
             theme = CONFIG_DEFAULTS.get('theme')
         self.__stylesheets.insert(1, 'themes/css/xtheme-%s.css' % theme)
 
+    def setup_webapidoc(self, enabled=True):
```

Comment:
I feel that enabling the docs should be read from config not passed as arg, thoughts?